lib/rules/no-duplicate-at-import-rules/README.md

```diff
@@ -0,0 +1,44 @@
+# no-duplicate-at-import-rules
+
+Disallow duplicate @import rules within a stylesheet.
+
+```css
+    @import "mystyle.css";@import "otherstyle.css";@import "mystyle.css";
+/**         ↑                                              ↑
+ * These are duplicates */
+```
+
+## Options
+
+### `true`
+
+The following patterns are considered violations:
+
+```css
```

Comment:
Please use `a.css` and `b.css` in examples and tests as that's a simple convention we've used elsewhere.